window.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2014-2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the license is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gowin32
+
+import (
+	"syscall"
+
+	"github.com/winlabs/gowin32/wrappers"
+)
+
+func GetWindowProcessID(hwnd syscall.Handle) (uint, error) {
+	var pid uint32
+	_, err := wrappers.GetWindowThreadProcessId(hwnd, &pid)
+	if err != nil {
+		return 0, NewWindowsError("GetWindowThreadProcessId", err)
```

Comment:
We can convert `ERROR_SUCCESS` to `EINVAL` here, since this one doesn't use 0 as a valid return value.

Reply:
You mean the conversion here in windows.go or rather in wrappers\winuser.go  ?

Reply:
My mistake.  I meant in wrappers/winuser.go.